examples/gaussian_processes/HSGP-Advanced.myst.md

```diff
@@ -0,0 +1,896 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: pymc-examples
+  language: python
+  name: pymc-examples
+---
+
+(hsgp)=
```

Comment:
hsgp is the key used to link to the HSGP-basic notebook, this needs to use something different

Reply:
Good catch @OriolAbril ! Updated